docs/Command-Blueprint.md

```diff
@@ -149,6 +163,49 @@ func verbNounRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 // verbNoun performs our actual function. It usually creates an API client,
 // configures it, configures an API request and performs it.
 func verbNoun(args verbNoundArguments) (verbNounResult, error) {
+  result := verbNounResult{}
+
+  // prepare client
+  clientConfig := client.Configuration{
+		Endpoint:  args.apiEndpoint,
+		Timeout:   10 * time.Second,
+		UserAgent: config.UserAgent(),
+	}
+  apiClient, clientErr := client.NewClient(clientConfig)
+	if clientErr != nil {
+		return result, microerror.Mask(couldNotCreateClientError)
+	}
+
+  authHeader := "giantswarm " + args.token
+	someResponse, rawResponse, err := apiClient.DoSomething(authHeader,
+		requestIDHeader, verbNounActivityName, cmdLine)
+
+  if rawResponse == nil || rawResponse.Response == nil {
+    return result, microerror.Mask(noResponseError)
+  }
+
+  // handle request errors
+  if err != nil {
+
+    switch rawResponse.StatusCode {
+    case http.StatusNotFound:
+      return result, microerror.Mask(clusterNotFoundError)
+    case http.StatusUnauthorized:
+      return result, microerror.Mask(notAuthorizedError)
+    case http.StatusForbidden:
+      return result, microerror.Mask(accessForbiddenError)
+    }
+
+		if rawResponse.StatusCode >= 500 {
+			return result, microerror.Maskf(internalServerError, err.Error())
+		}
+
+		return result, microerror.Mask(err)
+	}
+
+  // populate result base on some response information etc.
+  result.someAttribute = someResponse.someValue
 
+  return result, nil
 }
```

Comment:
`go fmt ./...`

Reply:
As this is a markdown file, that doesn't work

Reply:
```
// verbNoun performs our actual function. It usually creates an API client,
// configures it, configures an API request and performs it.
func verbNoun(args verbNoundArguments) (verbNounResult, error) {
	result := verbNounResult{}

	// prepare client
	clientConfig := client.Configuration{
		Endpoint:  args.apiEndpoint,
		Timeout:   10 * time.Second,
		UserAgent: config.UserAgent(),
	}
	apiClient, clientErr := client.NewClient(clientConfig)
	if clientErr != nil {
		return result, microerror.Mask(couldNotCreateClientError)
	}

	authHeader := "giantswarm " + args.token
	someResponse, rawResponse, err := apiClient.DoSomething(authHeader,
		requestIDHeader, verbNounActivityName, cmdLine)

	if rawResponse == nil || rawResponse.Response == nil {
		return result, microerror.Mask(noResponseError)
	}

	// handle request errors
	if err != nil {

		switch rawResponse.StatusCode {
		case http.StatusNotFound:
			return result, microerror.Mask(clusterNotFoundError)
		case http.StatusUnauthorized:
			return result, microerror.Mask(notAuthorizedError)
		case http.StatusForbidden:
			return result, microerror.Mask(accessForbiddenError)
		}

		if rawResponse.StatusCode >= 500 {
			return result, microerror.Maskf(internalServerError, err.Error())
		}

		return result, microerror.Mask(err)
	}

	// populate result base on some response information etc.
	result.someAttribute = someResponse.someValue

	return result, nil
}
```